cmd/generate/config/utils/generate.go

```diff
@@ -18,16 +18,17 @@ const (
 	identifierCaseInsensitivePrefix = `(?i:`
 	identifierCaseInsensitiveSuffix = `)`
 	identifierPrefix                = `(?:`
-	identifierSuffix                = `)(?:[0-9a-z\-_\t .]{0,20})(?:[\s|']|[\s|"]){0,3}`
+	identifierSuffix                = `)(?:[\w \t.-]{0,20})(?:\\*['"]{0,3})?[ \t]{0,5}`
 
 	// commonly used assignment operators or function call
-	operator = `(?:=|>|:{1,3}=|\|\|:|<=|=>|:|\?=)`
+	//language=regexp
+	operator = `(?:[<>?+]?=|:{1,3}=|>|=>|:|\(|,|\|)`
 
 	// boundaries for the secret
 	// \x60 = `
 	secretPrefixUnique = `\b(`
-	secretPrefix       = `(?:'|\"|\s|=|\x60){0,5}(`
-	secretSuffix       = `)(?:['|\"|\n|\r|\s|\x60|;]|$)`
+	secretPrefix       = `(?:\\*(?:'|")|[\x60\s=|>]){0,5}(`
```

Comment:
Since the quotes are a single char, could this just be?

```suggestion
	secretPrefix       = `(?:\\*[\'\"]|[\x60\s=|>]){0,5}(`
```

And just confirming the `|` is an intentional char match in `[\x60\s=|>]`, correct?

Reply:
You're right. I believe `(?:...)` is a remnant of an earlier change.

And yes, `|` is intentional.